src/ngraph/runtime/gpu/gpu_util.cpp

```diff
@@ -77,3 +77,101 @@ void runtime::gpu::cuda_memset(void* dst, int value, size_t buffer_size)
 {
     cudaMemset(dst, value, buffer_size);
 }
+
+namespace
+{
+    uint64_t _powU64(uint64_t base, uint64_t exp)
```

Comment:
can this be shared for all platform?

Reply:
I \*believe\* so. Any particular concern with it? We can add additional reviewers if it would help

Reply:
Just think if this will benefit other platform.